pythclient/calendar.py

```diff
@@ -47,19 +47,19 @@ def is_market_open(asset_type: str, dt: datetime.datetime) -> bool:
             if (
                 date in EQUITY_EARLY_HOLIDAYS
                 and time >= EQUITY_OPEN
-                and time <= EQUITY_EARLY_CLOSE
+                and time < EQUITY_EARLY_CLOSE
             ):
                 return True
             return False
-        if day < 5 and time >= EQUITY_OPEN and time <= EQUITY_CLOSE:
+        if day < 5 and time >= EQUITY_OPEN and time < EQUITY_CLOSE:
             return True
         return False
 
     if asset_type in ["fx", "metal"]:
         if date in FX_METAL_HOLIDAYS:
             return False
         # On Friday the market is closed after 5pm
-        if day == 4 and time > FX_METAL_OPEN_CLOSE_TIME:
+        if day == 4 and time >= FX_METAL_OPEN_CLOSE_TIME:
```

Comment:
so what's the semantics if you query exactly at the close time? is the market closed then or open? (I ask because it's worth documenting that)